Gruntfile.js

```diff
@@ -10,8 +9,12 @@ module.exports = function( grunt ) {
 	const pkg = grunt.file.readJSON( "package.json" );
 	const pluginVersion = pkg.yoast.pluginVersion;
 
+	// Used to switch between development and release builds.
+	const developmentBuild = ! [ "release", "release:js", "artifact", "deploy:trunk", "deploy:master" ].includes( process.argv[ 2 ] );
+
 	// Define project configuration
 	var project = {
+		developmentBuild,
```

Comment:
This one has moved from `global.developmentBuild` to the  project variable. Is that intended and doesn't it have any consequences?

Reply:
Since the `developmentBuild` variable had to be changed anyway, I moved it to inside the function because it is also done that way in the gruntfile of `wordpress-seo`.